drupal/conf/site.yml

```diff
@@ -22,6 +22,7 @@ default:
     create:
```

Comment:
Should there also be some kind of core-update command now that would take care of updating core, scaffold and such?

Reply:
In create command there is this scaffold command which downloads the scaffold: `shell: composer drupal-scaffold`